src/ideation/ideation.service.ts

```diff
@@ -63,11 +74,36 @@ export class IdeationService {
         return query.map((q) => Object.values(q).flat()).flat();
     }
 
-    // update(id: number, updateIdeationDto: UpdateIdeationDto) {
-    //     return `This action updates a #${id} ideation`;
-    // }
+    async update(
+        id: number,
+        pId: number,
+        updateIdeationDto: UpdateIdeationDto,
+    ) {
+        return this.prisma.projectIdea.update({
+            where: {
+                id: pId,
+            },
+            data: updateIdeationDto,
+        });
+    }
 
-    // remove(id: number) {
-    //     return `This action removes a #${id} ideation`;
-    // }
+    async remove(id: number, pId: number) {
```

Comment:
we need a check where only the user submitted the idea can remove it, Although, they shouldn't be able to remove if other people have also voted for it (because obviously they also like the idea).

Reply:
Do we have a way to verify who is logged in? I thought this upcoming sprint is when we are adding authentication.

Reply:
Not currently. We could do a temporary workaround by taking the userid as a parameter and just picking one from db to send from the frontend. The permanent solution should be making use of the auth and auth token to verify the user.